Makefile

```diff
@@ -71,7 +71,9 @@ smithy-go-publish-local:
 
 gen-config-asserts:
 	@echo "Generating SDK config package implementor assertions"
-	cd config && go generate
+	cd config \
+	    && go mod tidy \
```

Comment:
does mod tidy need to be mixed into these tasks? mod tidy has a separate task that will be run by make generate.

Reply:
We have to run go mod tidy, when we cd into Config. Or we get an error related to go sum entry not found.

Reply:
Ah i see, could also set `GOFLAGS=-mod=mod` here as well potentially instead of running mod tidy.